python/cxx/parser.cpp

```diff
@@ -60,7 +62,8 @@ void opmcommon_python::export_Parser(py::module& module) {
 
 
     py::class_<ParserKeyword>(module, "ParserKeyword")
-        .def_property_readonly("name", &ParserKeyword::getName);
+        .def_property_readonly("name", &ParserKeyword::getName)
+        .def("create_deckkeyword", &create_deckkeyword, ref_internal);
```

Comment:
Is there a reason you did not bind this to the `DeckKeyword` constructor the normal way?

Reply:
DeckKeyword has an internal pointer to a ParserKeyword, which need to be kept alive.